packages/backend-data/src/convert_js_resolvers.ts

```diff
@@ -0,0 +1,81 @@
+import { Construct } from 'constructs';
+import { AmplifyData } from '@aws-amplify/data-construct';
+import { readFileSync } from 'fs';
+import * as appsync from 'aws-cdk-lib/aws-appsync';
+// TODO: uncomment after data-schema-types change is published
+// import { JsResolver } from '@aws-amplify/data-schema-types';
+
+// Temporarily adding this type for draft PR. Will be imported from data-schema-types
+// once that package is updated with this new type
+export type JsResolver = {
+  typeName: 'Mutation' | 'Query' | 'Subscription';
+  fieldName: string;
+  handlers: {
+    dataSource: string;
+    entry: string;
```

Comment:
are we going to support any sort of build step for these files? or does it have to be raw js?

Reply:
However, what should we keep in mind if we decide we want to add a build step in the future?

For example, we may want to ensure the `response` handler return type conforms to the return type configured in the schema definition. (We might look into this post-GA.)